guvnor-webapp/pom.xml

```diff
@@ -666,6 +666,7 @@
             <compileSourcesArtifact>org.kie.uberfire:kie-uberfire-social-activities-api</compileSourcesArtifact>
 
             <!-- Dashbuilder -->
+            <compileSourcesArtifact>org.dashbuilder:dashbuilder-json</compileSourcesArtifact>
```

Comment:
Ah, my bad. It seems to work even without adding to project dependencies. My failure was caused by something else.